tidy3d/components/data/sim_data.py

```diff
@@ -213,13 +213,103 @@ def at_centers(self, field_monitor_name: str) -> xr.Dataset:
 
         return monitor_data.colocate(**xyz_kwargs)
 
-    def get_intensity(self, field_monitor_name: str) -> xr.DataArray:
+    def get_poynting_vector(self, field_monitor_name: str) -> xr.Dataset:
+        """return `xarray.Dataset` of the Poynting vector at Yee cell centers.
```

Comment:
for RST, we should use double backslash here.

Reply:
same below (and might be good to put the math bit `S = 1/2 E × conj(H)` in double backslash too